### ISSUE TITLE
Merging of integrated sample or merged sample?

### ISSUE THREAD

**Issue by EmanuelSoda** [satijalab/seurat#7347]
Hi satijalab, 

First of all thanks for the fantastic package. 
I have a question regarding integration and merging. I have three different target cells in three distinct Seurat objects. 
For those, I have WT and KI conditions. Initially, I analysed those objects individually by integrating WI and KI for each target cells. If I wanted to merge those objects together I can simply merge the integrated objects, then perform normalization and all the other steps on the merged integrated object or I should merge all samples at the beginning and then integrate for WT and KI? 

Basically, my question boils down to "integrating merged objects is the same as merging integrated objects?"

Thanks in advance, 
Emanuel

**Comment by nathan-nakatsuka** [comment]
Dear @EmanuelSoda , merging integrated objects I believe would not be the same, since they will have a different set of anchors than if you integrate after merging all of them. You should try visualizing the answers to both of them and they likely will be a bit different. You should generally merge and then integrate afterwards. However, you can integrate then merge only if you perform SCT normalization and find integration features and then merge the normalized Seurat objects. This might be helpful: https://hbctraining.github.io/scRNA-seq_online/lessons/06a_integration_harmony.html.
